web/packages/teleport/src/components/FormLogin/FormLogin.tsx

```diff
@@ -31,6 +30,7 @@ import {
   Text,
 } from 'design';
 import * as Alerts from 'design/Alert';
+import { Divider } from 'design/Divider';
```

Comment:
I think the Divider component changes belongs to different PR, there's already large change related to supporting checkboxes.

Reply:
OK, I'll just copy it over to the new place.